accounts/keystore/keystore_test.go

```diff
@@ -462,10 +448,7 @@ func checkEvents(t *testing.T, want []walletEvent, have []walletEvent) {
 }
 
 func tmpKeyStore(t *testing.T, encrypted bool) (string, *KeyStore) {
```

Comment:
We don't need to return the dir here anymore

Reply:
We still need to return the `dir` here. It is being used in two tests: https://github.com/ethereum/go-ethereum/pull/24633#discussion_r841953459 and https://github.com/ethereum/go-ethereum/pull/24633#discussion_r841953523